### DISCUSSION TITLE
Reconsider Explore Recommendations icons

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#50763] [Mobile] [answered]
GitHub Mobile beta

1. Browse to explore 🔭
2. Edit filters
    ![Screenshot_20230322-222649.png](https://user-images.githubusercontent.com/2119212/227084447-0bdde9ea-e224-48c5-9e00-456a0bf99e19.png)
    Note that the icon for recommendations is the octocat. 
3. There are two kinds of recommendations 
    ![Screenshot_20230322-222638.png](https://user-images.githubusercontent.com/2119212/227084460-eed17f31-650c-42ad-b23c-22cf1ab83214.png)
    🔭👤

These icons don't really fit well.

The entire explore category is 🔭. The section is GitHub. Some items are 🔭 and others are 👤. But, GitHub is doing all the suggestions, so I don't think that's a great icon to use, and the sandwich of 🔭 is awkward. The opposite of 👤 would be either 🌐 or GitHub. Recommendations could be 🪄.

**Comment by aconsuegra** [comment, marked as answer]
Thanks for your feedback @jsoref! The Mobile apps are using the same icon that we see in github.com, so this should be a global change across all the platforms. I'll create an issue for our design team with your feedback. Thanks again!
